proposals/thread-spawn/Overview.md

```diff
@@ -1,11 +1,13 @@
 # [DRAFT] Thread Spawn Proposal
 
 This page describes a proposal to allow WebAssembly modules to spawn threads from within the
-WebAssembly language. This is done with three additions to the specification:
+WebAssembly language. This is done with several additions to the specification (each addition is
+annotated with its criticality to the proposal; next-stage discussions could alter this list):
 
-- a new `thread.spawn` instruction
-- new `shared` attributes on WebAssembly tables, functions, and globals
-- a new `thread.hw_concurrency` instruction
+- __required__: new `shared` attributes on WebAssembly tables, functions, and globals
+- __required in some form__: a new `thread.spawn` instruction (other mechanisms are possible; more
+  discussion needed)
```

Comment:
would mention here (less fundamental than `thread.spawn/id`, more fundamental than `hw_concurrency`) that we need a story for JS interaction - either working through how dispatch threads work (this becomes very complicated if `thread.id` is a JS host call rather than a real instruction) or potentially adding some new capability to the JS api

Reply:
See my very vague addition: "a design for JavaScript-WebAssembly interaction." There's a lot more that can be said but at this point in the overview it might be good to leave it vague.